src/store.ts

```diff
@@ -11,6 +11,7 @@ export class Normi {
     this.params = {
       id: params.id || ['id'],
     };
+    makeObservable(this);
```

Comment:
Instead, I recommend just using `makeAutoObservable` and getting rid of all the decorators in the class definition. When I wrote `normi` initially, makeAutoObservable didn't exist.